src/Utils/resizer.ts

```diff
@@ -1,14 +1,14 @@
-import qs from "qs"
+import { configure } from "@artsy/img"
 import { getENV } from "./getENV"
 
 export const GEMINI_CLOUDFRONT_URL =
   getENV("GEMINI_CLOUDFRONT_URL") ?? "https://d7hftxdivxxvm.cloudfront.net"
 
-const warn = (message: string) => {
-  if (process.env.NODE_ENV === "development") {
-    console.warn(message)
-  }
-}
+const services = configure({
```

Comment:
So the ultimate plan here is to configure multiple services, then switch between them while noting the service as a tag to the end user metrics and passing along the requested service to Metaphysics for its URL generation, so that we're loading images consistently for a single session.

Reply:
Cool yup, that sounds like the perfect plan, this looked like an easy drop-in.

I guess we also need to do this same thing in Metaphysics, and then there is a bit of intertwining where as we run experiments, we may need to coordinate setting  a corresponding ENV var in both Force and MP (or however we toggle experiments).

Random thought, is that compelling to use Unleash for (and possibly wrap that in the @artsy/img lib?)

Reply:
Yep, we'll integrate the same way in MP then we'll set a header in Relay that gets passed to Metaphysics to choose the service.

And yeah if we can use Unleash for this that'd be ideal. Force will drive the service selection, no need for the img lib to know anything about that.